### ISSUE TITLE
Support for motion sensors

### ISSUE THREAD

**Issue by nanderss** [slasktrat/com.cotech.433#1]
Are there any plans to add support for cotech motion sensors?

**Comment by slasktrat** [comment]
This one? https://www.clasohlson.com/uk/Motion-Detector/Pr361084000
It seems to me it is's only hard wired and do not send any 433Mhz or other wireless signal?

**Comment by nanderss** [comment]
No, it's this one: https://m.clasohlson.com/uk/Cotech-Smart-Home-PIR-Sensor/36-6362
